app.py

```diff
@@ -52,7 +52,7 @@ def experience():
     Handle experience requests
     '''
     if request.method == 'GET':
-        return jsonify()
+        return jsonify(data['experience'])
```

Comment:
Is there a unit test for this as well? If not, it might be a good idea to add it in this PR so that we know things are working perfectly

Reply:
There is a unit test in the file `test_pytest.py` that tests this feature. That specific test should pass now because the POST request functionality has been implemented.